websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "658765364439810048"
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/yAhhPzj.png",
+			details: "Exploring",
+			startTimestamp: Date.now(),
+			buttons: [
+				{
+					label: "View Page",
+					url: location.href
+				}
+			]
+		},
+		page = location.href.split("#")[0].split("?")[0].split("/").slice(-1);
```

Comment:
I'm not sure why this is so complicated, maybe using [SearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams) my help?